### DISCUSSION TITLE
Error after deploying blog template through sanity

### DISCUSSION THREAD

**Discussion by mikohargett** [vercel/community#4286] [Help]
### Summary

I keep getting the following error after deploying the next.js blog template from Sanity when it's preparing the Git repository. I checked the status page but saw no updates related to this. Has anyone run into this or know what to do?

An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.
![image](https://github.com/vercel/community/assets/77860706/1be2a438-07f8-4ac3-8fc2-31c171480375)

### Steps to Reproduce

I went to this page and clicked Deploy. 
https://www.sanity.io/templates/blog-with-built-in-content-editing

Clicked Deploy again on this page: 
https://vercel.com/templates/next.js/blog-nextjs-sanity

Got to this page, https://vercel.com/new/clone?demo

Created a git repository, and added the sanity integration successfully. 

Then I got the error. 
![image](https://github.com/vercel/community/assets/77860706/69d6e777-80eb-4dd8-96eb-84be865280e1)

**Comment by amyegan** [comment]
Thanks for sharing the template. I didn't get an error when deploying it just now. Are you still getting this error?
